modules/xmpp/JingleSessionPC.js

```diff
@@ -926,15 +926,19 @@ export default class JingleSessionPC extends JingleSession {
      * added, before the offer/answer cycle executes (for the local track
      * addition to be an atomic operation together with the offer/answer).
      */
-    invite(localTracks) {
+    invite(localTracks = []) {
         if (!this.isInitiator) {
             throw new Error('Trying to invite from the responder session');
         }
         const workFunction = finishedCallback => {
+            const addTracks = [];
+
             for (const localTrack of localTracks) {
                 this.peerconnection.addTrack(localTrack, true /* isInitiator */);
```

Comment:
I think you are missing the push to `addTracks` here.

Reply:
🤦